README.md

```diff
@@ -111,3 +116,19 @@ note ={PMID: 33155465},
 URL = {https://doi.org/10.1021/acs.jcim.0c00839}
 }
 ```
+## List of publications
+- https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c00608 
+- https://www.sciencedirect.com/science/article/pii/S2001037023003495
+
+- https://doi.org/10.1021/acs.jmedchem.1c01803 ?
+- https://10.1021/acs.jmedchem.2c00931 ?
+- https://pubs.acs.org/doi/full/10.1021/acs.jcim.1c00630 ?
+- https://www.sciencedirect.com/science/article/pii/S1359644622001337 ?
+- https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c00050 (does not use kinfraglib but uses methodology) ?
+
+- https://chemrxiv.org/engage/chemrxiv/article-details/658441f7e9ebbb4db96d98e8
+- https://chemrxiv.org/engage/chemrxiv/article-details/60c759054c89195d4ead4ccd
```

Comment:
Skip this, the preprint has been published here: https://pubs.acs.org/doi/10.1021/acs.jcim.1c00608

Reply:
Both? ( https://chemrxiv.org/engage/chemrxiv/article-details/658441f7e9ebbb4db96d98e8
- https://chemrxiv.org/engage/chemrxiv/article-details/60c759054c89195d4ead4ccd)

Reply:
No, keep this one: https://chemrxiv.org/engage/chemrxiv/article-details/658441f7e9ebbb4db96d98e8, remove this: https://chemrxiv.org/engage/chemrxiv/article-details/60c759054c89195d4ead4ccd and add the published version (https://pubs.acs.org/doi/10.1021/acs.jcim.1c00608) if it's not in the list already

Reply:
But (https://pubs.acs.org/doi/10.1021/acs.jcim.1c00608) is not the published version of https://chemrxiv.org/engage/chemrxiv/article-details/658441f7e9ebbb4db96d98e8?
https://chemrxiv.org/engage/chemrxiv/article-details/658441f7e9ebbb4db96d98e8: Is this already pushed? I cannot find a non-preprint version

Reply:
Okay wait maybe I mixed up links in the previous comment, so keep the preprint paper by Michael and Joschka (called guided docking... ) in the list since it's not published anywhere yet. Then, remove this preprint paper (title: A Deep Learning Based Scaffold Hopping Strategy for the Design of Kinase Inhibitors), since this has been published (and renamed) in JCIM (title: Kinase Inhibitor Scaffold Hopping with Deep Learning Approaches, add this).